py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/s3_utils.py

```diff
@@ -36,3 +38,35 @@ def invoke_async_lambda(function_arn: str, event: dict) -> None:
     lambda_client.invoke(FunctionName=function_arn,
                          InvocationType='Event',
                          Payload=json.dumps(event))
+
+def move_3s_objects(obj_list: list[str], 
+                    src_bucket: str,
+                    dest_bucket: str) -> None:
+    """
+    Move list of S3 objects from src_bucket to dest_bucket.
+
+    :param obj_list: list of S3 object paths (paths must start with src_bucket)
+    :param src_bucket: S3 bucket to move from
+    :param dest_bucket: S3 bucket to move to
+
+    No return value.
+    """
+    s3_client = get_s3_client()
+    for obj in obj_list:
+        # Check if source bucket is in obj path, should skip local files???
+        if src_bucket not in obj:
+            logging.warn(f"{src_bucket} bucket not found in {obj} path")
```

Comment:
i've been using string insertion via `%` for my logging statements.

i don't have really strong preferences about which we use, but we should be consistent.